dagobah/core/job.py

```diff
@@ -439,8 +439,11 @@ def _serialize(self, include_run_logs=False, strict_json=False):
             result = json.loads(json.dumps(result, cls=StrictJSONEncoder))
         return result
 
-    def _implements_function(obj, functions):
+    def _implements_function(self, obj, functions):
         """ Checks for the existence of a method or list of methods """
+        if isinstance(functions, str):
+            functions = [functions]
+        logger.debug("Checking if {0} implements {1}".format(obj, functions))
```

Comment:
Probably not, there was even more before for my own purposes while unit testing initially, I'll take this out.